lib/mruby/build/command.rb

```diff
@@ -126,6 +127,31 @@ def define_rules(build_dir, source_dir='')
       end
     end
 
+    def compiles?(source_text)
```

Comment:
@matz I hope this is sufficient for what you were thinking about adding autoconf like behavior when we need.